symphony/content/content.blueprintspages.php

```diff
@@ -649,7 +648,6 @@ public function __actionEdit()
 
             if (trim($fields['handle']) == '') {
                 $fields['handle'] = $fields['title'];
-                $autogenerated_handle = true;
```

Comment:
But this scenario is when `$autogenerated_handle` is set to `true`, not `false` so although rare, it _is_ possible?